client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { includes } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getJetpackSettingsSaveError, getJetpackSettingsSaveRequestStatus } from 'state/selectors';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+const SpamFilteringSettings = ( {
+	fields,
+	dirtyFields,
+	hasAkismetKeyError,
+	isRequestingSettings,
+	isSavingSettings,
+	onChangeField,
+	translate
+} ) => {
+	const { wordpress_api_key, akismet: akismetActive } = fields;
+	const isDirty = includes( dirtyFields, 'wordpress_api_key' );
+	const header = ( akismetActive && wordpress_api_key && ! isDirty )
+		? translate( 'Your site is protected from spam.' )
+		: translate( 'Your site needs an Antispam key.' );
+
+	const inTransition = isRequestingSettings || isSavingSettings;
+	const validationText = hasAkismetKeyError
+		? translate( 'There\'s a problem with your Antispam API key.' )
+		: translate( 'Your Antispam key is valid.' );
+	const className = ( inTransition || isDirty )
+		? null
+		: classNames( {
+			'is-valid': wordpress_api_key && ! hasAkismetKeyError,
+			'is-error': wordpress_api_key && hasAkismetKeyError
+		} );
+	return (
+		<div className="site-settings__spam-filtering">
+			<FoldableCard header={ header }>
+				<FormFieldset>
+					<div className="spam-filtering-settings__info-link-container site-settings__info-link-container">
+						<InfoPopover >
+							<ExternalLink target="_blank" icon
+								href={ 'https://jetpack.com/features/security/spam-filtering/' }>
```

Comment:
Nitpicking, but I prefer closing the `>` on a separate line, with one tab to the left. This way you can see where the opening tag ends. Otherwise the eye might think that the `{ translate...` is part of a prop you're passing there.